src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -52,36 +52,12 @@ export function transform(formConfig, form) {
   });
 }
 
+export function transformDisabilities(disabilities) {
+  return disabilities.map(disability => set('disabilityActionType', 'INCREASE', disability));
+}
 
-export const isPrefillDataComplete = (formData) => {
-  const { socialSecurityNumber, vaFileNumber, gender,
-    dateOfBirth, servicePeriods } = formData;
-  const first = get('fullName.first', formData);
-  const last = get('fullName.last', formData);
-  const country = get('veteran.mailingAddress.country', formData);
-  const addressLine1 = get('veteran.mailingAddress.addressLine1', formData);
-  const emailAddress = get('veteran.emailAddress', formData);
-  const primaryPhone = get('veteran.primaryPhone', formData);
-  const accountType = get('directDeposit.accountType', formData);
-  const routingNumber = get('directDeposit.routingNumber', formData);
-  const bankName = get('directDeposit.bankName', formData);
-  const noBank = get('directDeposit.noBank', formData);
-  const hasVeteranDetails = first && last && gender && dateOfBirth && (socialSecurityNumber || vaFileNumber);
-  const hasPrimaryAddressInfo = country && addressLine1 && emailAddress && primaryPhone;
-  const hasPaymentInfo = noBank || (accountType && routingNumber && bankName);
-  const hasMilitaryHistoryInfo = servicePeriods && servicePeriods.length > 0;
-  return !!(hasVeteranDetails && hasPrimaryAddressInfo && hasPaymentInfo && hasMilitaryHistoryInfo);
-};
-
-
-export function prefillTransformer(pages, formData, metadata, state) {
-  let newData = formData;
-  const isPrefilled = state.prefilStatus === PREFILL_STATUSES.success;
-  const hasRequiredInformation = isPrefillDataComplete(formData);
-
-  if (isPrefilled && hasRequiredInformation) {
-    newData = set('prefilled', true, newData);
-  }
+export function prefillTransformer(pages, formData, metadata) {
+  const newData = set('disabilities', transformDisabilities(formData.disabilities), formData);
```

Comment:
It might also be good to check that all the disabilities are valid after this and throw an error to sentry if they aren't.

Sort of out of scope for the connected ticket, perhaps, but it's close enough (and the PR small enough) that it might make enough sense. :man_shrugging:

Reply:
Got it, thanks for clarifying, will add the validation back in.

Reply:
Sure thing! Just to be clear, I mean only for logging to sentry. We shouldn't filter the list yet.